migrations/versions/6b2e5cfdd0f0_.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
```

Comment:
Delete the empty migration

Reply:
@iamareebjamal then it will show error for multiple heads.

Reply:
Use the merge head commands

Reply:
This files is created after I ran the merge head command.